tests/django_restframework_gis_tests/test_filters.py

```diff
@@ -71,6 +73,7 @@ def test_inBBOXFilter_filtering(self):
         for result in response.data['features']:
             self.assertEqual(result['properties']['name'] in ('isContained', 'isEqualToBounds', 'overlaps'), True)
 
+    @skipIf(settings.DATABASES['default']['ENGINE'] == 'django.contrib.gis.db.backends.spatialite', 'Spatialite detected')
```

Comment:
since `settings.DATABASES['default']['ENGINE'] == 'django.contrib.gis.db.backends.spatialite'` and 'Spatialite detected' are repeated 3 times, could you define them once at the beginning?

Could you write just a more self explainatory message as a help for the maintainers that will come here in the future? Something like "Skipped test for spatialite backend: missing feature" or anything better you can come up with.